pkg/tests/end_to_end_tests/promql_query_endpoint_test.go

```diff
@@ -2506,6 +2506,10 @@ func TestPromQLQueryEndpoint(t *testing.T) {
 			name:  "two pushdowns, same metric different matchers",
 			query: `sum(rate(metric_2{foo = "bar"}[5m]))/sum(rate(metric_2[5m]))`,
 		},
+		{
+			name:  "one pushdown, matrix",
```

Comment:
Can you maybe add some tests with offset?

Reply:
Can you be more specific? There are already quite a few tests which use offsets in `promql_query_endpoint_test.go`. Is there a specific case that you would like to exercise?

Reply:
I was thinking something like: `rate(metric_2[1m])/metric_2 offset 2m`

Reply:
done